src/Codeception/Test/Interfaces/ScenarioDriven.php

```diff
@@ -1,16 +1,16 @@
 <?php
+
 namespace Codeception\Test\Interfaces;
 
+use Codeception\Scenario;
+
 interface ScenarioDriven
 {
     public function getFeature();
 
-    /**
-     * @return \Codeception\Scenario
-     */
-    public function getScenario();
+    public function getScenario(): Scenario;
 
-    public function getScenarioText($format = 'text');
+    public function getScenarioText(string $format = 'text');
```

Comment:
`getScenarioText` and `getSourceCode` return strings, `preload` returns void and I haven't found any cases where feature is not string, so I think that `getFeature` returns `string` or `?string`.